draft-ietf-taps-interface.md

```diff
@@ -1290,6 +1293,19 @@ Size for the Connection, the Send will fail with a SendError event ({{send-error
 example, it is invalid to send a Message over a UDP connection that is larger than
 the available datagram sending size.
 
+## Sending Replies {#send-replies}
+
+When a message is sent in response to a message received, the application
+may use the Message Context of the received Message to construct a Message Context for the reply.
+
+~~~
+replyMessgeContext := replyMessageContext.reply()
```

Comment:
Typo: `replyMessgeContext` -> `replyMessageContext`

Reply:
Also, it feels odd to have replyMessageContext come from the reply of itself. Should these be two separate variables?

Reply:
Sorry - It seems it was way too hot (40°C+) in my office… 
```replyMessageContext := requestMessageContext.reply()```
fixed